src/OrchardCore.Modules/OrchardCore.Menu/Views/MenuItemLink-HtmlMenuItem.cshtml

```diff
@@ -17,6 +17,12 @@
     }
 
     tag.Attributes["href"] = url;
+
+    if (!string.IsNullOrEmpty(htmlMenuItemPart.Target))
+    {
+        tag.Attributes["target"] = htmlMenuItemPart.Target;
```

Comment:
Validate the value to make sure only valid values can be provided

Reply:
This is really not necessary, since the target attribute is freely definable, e.g. target="order_details" , and when the user clicks on a link it will always refresh the existing window as much as possible, instead of always opening a new one.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#target